frontend/src/components/ItemTable.tsx

```diff
@@ -0,0 +1,54 @@
+import {ItemModel} from "../model/ItemModel";
+
+import "../css/ItemTable.css"
+import ItemTableRow from "./ItemTableRow";
+
+type itemPageProbs = {
+    data: ItemModel[],
+    fetchData: () => void
+    setErrorModal: (showAlert: boolean) => void
+    setErrorMessage: (errorMessage: string) => void
+    setSuccessMessage: (successMessage: string) => void
+    setSuccessModal: (showSuccessAlert: boolean) => void
+}
+
+function ItemTable(props: itemPageProbs) {
+
+    return (
+        <div className="container-table">
+            <table className="blueTable">
+                <thead>
+                <tr>
+                    <th>item-Nr</th>
+                    <th>ean</th>
+                    <th>name</th>
+                    <th>category</th>
+                    <th>issuing country</th>
+                    <th>capacity</th>
+                    <th>action</th>
+                </tr>
+                </thead>
+                <tbody>
+                {props.data.length === 0 ? (
+                    <tr>
+                        <td>data not found</td>
+                        <td>data not found</td>
+                        <td>data not found</td>
+                        <td>data not found</td>
+                        <td>data not found</td>
+                        <td>data not found</td>
```

Comment:
Oder eine Zelle, die sich über alle Spalten erstreckt?
```suggestion
                        <td colspan="6">data not found</td>
```